Neos.Flow/Scripts/migrate.php

```diff
@@ -46,11 +46,18 @@
 
 $migrationsManager = new Manager();
 
-$packageKey = getFlagValue('package-key');
+if (!isset($GLOBALS['argv'][0]) || substr($GLOBALS['argv'][0], 0, 2) === '--') {
+    outputLine('EXCEPTION: No package key specified.', [], 0, STYLE_ERROR);
+    outputLine('  That package key has to be specified as first argument like "./flow flow:core:migrate Foo.Bar:MyPackage"', [], 0, STYLE_ERROR);
+
```

Comment:
The package key argument needs to be specified before any flags because otherwise we'd need to implement the whole flag vs argument logic which is not trivial..